README.md

```diff
@@ -28,7 +28,7 @@ Currently supports:
 | `transformInclude`<sup>2</sup> | ✅ | ✅ | ✅ | ✅ | ✅ |
 | [`transform`](https://rollupjs.org/guide/en/#transformers) | ✅ | ✅ | ✅ | ✅ | ✅ <sup>3</sup> |
 | [`watchChange`](https://rollupjs.org/guide/en/#watchchange) | ✅ | ✅ | ✅ | ✅ | ✅ |
-| [`buildEnd`](https://rollupjs.org/guide/en/#buildend) | ✅ | ✅ | ✅ | ✅ | ✅ |
```

Comment:
Hi, is removing build end support here intentional?

Reply:
Oh, it was a mistake.

Reply:
fixed

Reply:
Awesome, thank you!